client/src/pages/Profile.js

```diff
@@ -1,71 +1,98 @@
-import React, { useState } from 'react';
-import Button from '@material-ui/core/Button';
-import Following from '../Components/Following';
-import { makeStyles } from '@material-ui/core';
+import React, { useState, useEffect } from "react";
+import Button from "@material-ui/core/Button";
+import Following from "../components/Following";
+import { makeStyles } from "@material-ui/core";
 // import photo1 from '../../assets/a0ebf9987c35f57f8bb9c8639b3a67fbd40ddaef.png'
-import './Profile.css';
+import "./Profile.css";
+import { type } from "os";
```

Comment:
Is this unused?

Reply:
I'm not using it.  It might be in the plans for something Badr or Shojol were contributing.  I plan a big cleanup/tidying when I reach a milestone.